dev/ci/scripts/unittests/test_create_experiment.py

```diff
@@ -31,3 +31,7 @@ def test_create_experiment():
     assert err == 0, f"create_experiment.py failed!"
 
     rmtree(RUNDIR_FAKE)
+
+
+def test_something():
+    assert 1 == 1
```

Comment:
```suggestion
```
remnant, probably for debugging.

Reply:
Oh yeah, thought I had removed that alread.